<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Joana L. Rocha</title> <meta name="author" content="Joana L. Rocha"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.4.0/css/all.min.css" integrity="sha256-HtsXJanqjKTc8vVQjO4YMhiqFoXkfBsjBWcX91T1jr8=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://joanocha.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Joana </span>L. Rocha</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">research</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="makova_complete_2024" class="col-sm-8"> <div class="title">The complete sequence and comparative analysis of ape sex chromosomes</div> <div class="author"> Kateryna D. Makova, and <span class="more-authors" title="click to view 83 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '83 more authors' ? 'Brandon D. Pickett, Robert S. Harris, Gabrielle A. Hartley, Monika Cechova, Karol Pal, Sergey Nurk, DongAhn Yoo, Qiuhui Li, Prajna Hebbar, Barbara C. McGrath, Francesca Antonacci, Margaux Aubel, Arjun Biddanda, Matthew Borchers, Erich Bornberg-Bauer, Gerard G. Bouffard, Shelise Y. Brooks, Lucia Carbone, Laura Carrel, Andrew Carroll, Pi-Chuan Chang, Chen-Shan Chin, Daniel E. Cook, Sarah J. C. Craig, Luciana Gennaro, Mark Diekhans, Amalia Dutra, Gage H. Garcia, Patrick G. S. Grady, Richard E. Green, Diana Haddad, Pille Hallast, William T. Harvey, Glenn Hickey, David A. Hillis, Savannah J. Hoyt, Hyeonsoo Jeong, Kaivan Kamali, Sergei L. Kosakovsky Pond, Troy M. LaPolice, Charles Lee, Alexandra P. Lewis, Yong-Hwee E. Loh, Patrick Masterson, Kelly M. McGarvey, Rajiv C. McCoy, Paul Medvedev, Karen H. Miga, Katherine M. Munson, Evgenia Pak, Benedict Paten, Brendan J. Pinto, Tamara Potapova, Arang Rhie, Joana L. Rocha, Fedor Ryabov, Oliver A. Ryder, Samuel Sacco, Kishwar Shafin, Valery A. Shepelev, Viviane Slon, Steven J. Solar, Jessica M. Storer, Peter H. Sudmant, Sweetalana, Alex Sweeten, Michael G. Tassia, Françoise Thibaud-Nissen, Mario Ventura, Melissa A. Wilson, Alice C. Young, Huiqing Zeng, Xinru Zhang, Zachary A. Szpiech, Christian D. Huber, Jennifer L. Gerton, Soojin V. Yi, Michael C. Schatz, Ivan A. Alexandrov, Sergey Koren, Rachel J. O’Neill, Evan E. Eichler, Adam M. Phillippy' : '83 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">83 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Jun 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-024-07473-2"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-024-07473-2" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Reference assemblies of great ape sex chromosomes show that Y chromosomes are more variable in size and sequence than X chromosomes and provide a resource for studies on human evolution and conservation genetics of non-human apes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/SVreview_fig3-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/SVreview_fig3-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/SVreview_fig3-1400.webp"></source> <img src="/assets/img/publication_preview/SVreview_fig3.jpeg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="SVreview_fig3.jpeg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="l_rocha_structural_2024" class="col-sm-8"> <div class="title">Structural variation in humans and our primate kin in the era of telomere-to-telomere genomes and pangenomics</div> <div class="author"> <em>Joana L. Rocha*</em>, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Runyang N Lou*, Peter H Sudmant' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Current Opinion in Genetics &amp; Development</em>, Aug 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.gde.2024.102233"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.gde.2024.102233" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Structural variants (SVs) account for the majority of base pair differences both within and between primate species. However, our understanding of inter- and intra-species SV has been historically hampered by the quality of draft primate genomes and the absence of genome resources for key taxa. Recently, advances in long-read sequencing and genome assembly have begun to radically reshape our understanding of SVs. Two landmark achievements include the publication of a human telomere-to-telomere (T2T) genome as well as the development of the first human pangenome reference. In this review, we first look back to the major works laying the foundation for these projects. We then examine the ways in which T2T genome assemblies and pangenomes are transforming our understanding of and approach to primate SV. Finally, we discuss what the future of primate SV research may look like in the era of T2T genomes and pangenomics.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/research_briefing_figure-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/research_briefing_figure-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/research_briefing_figure-1400.webp"></source> <img src="/assets/img/publication_preview/research_briefing_figure.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="research_briefing_figure.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="bolognini_global_2024" class="col-sm-8"> <div class="title">Global diversity, recurrent evolution, and recent selection on amylase structural haplotypes in humans</div> <div class="author"> Davide Bolognini*, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Alma Halgren*, Runyang Nicolas Lou*, Alessandro Raveane*, Joana L. Rocha*, Andrea Guarracino, Nicole Soranzo, Jason Chin, Erik Garrison, Peter H. Sudmant' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> Jun 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2024.02.07.579378"></span> <span class="__dimensions_badge_embed__" data-doi="10.1101/2024.02.07.579378" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The adoption of agriculture, first documented ∼12,000 years ago in the Fertile Crescent, triggered a rapid shift toward starch-rich diets in human populations. Amylase genes facilitate starch digestion and increased salivary amylase copy number has been observed in some modern human populations with high starch intake, though evidence of recent selection is lacking. Here, using 52 long-read diploid assemblies and short read data from ∼5,600 contemporary and ancient humans, we resolve the diversity, evolutionary history, and selective impact of structural variation at the amylase locus. We find that amylase genes have higher copy numbers in populations with agricultural subsistence compared to fishing, hunting, and pastoral groups. We identify 28 distinct amylase structural architectures and demonstrate that nearly identical structures have arisen recurrently on different haplotype backgrounds throughout recent human history. AMY1 and AMY2A genes each exhibit multiple duplications/deletions with mutation rates \textgreater10,000-fold the SNP mutation rate, whereas AMY2B gene duplications share a single origin. Using a pangenome graph-based approach to infer structural haplotypes across thousands of humans, we identify extensively duplicated haplotypes present at higher frequencies in modern day populations with traditionally agricultural diets. Leveraging 533 ancient human genomes we find that duplication-containing haplotypes (i.e. haplotypes with more amylase gene copies than the ancestral haplotype) have increased in frequency more than seven-fold over the last 12,000 years providing evidence for recent selection in West Eurasians. Together, our study highlights the potential impacts of the agricultural revolution on human genomes and the importance of long-read sequencing in identifying signatures of selection at structurally complex loci.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/fennec_cover-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/fennec_cover-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/fennec_cover-1400.webp"></source> <img src="/assets/img/publication_preview/fennec_cover.jpeg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="fennec_cover.jpeg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="l_rocha_north_2023" class="col-sm-8"> <div class="title">North African fox genomes show signatures of repeated introgression and adaptation to life in deserts</div> <div class="author"> <em>Joana L. Rocha</em>, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Pedro Silva, Nuno Santos, Mónia Nakamura, Sandra Afonso, Abdeljebbar Qninba, Zbyszek Boratynski, Peter H. Sudmant, José C. Brito, Rasmus Nielsen, Raquel Godinho' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>Nature Ecology &amp; Evolution</em>, Jun 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s41559-023-02094-w" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41559-023-02094-w"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41559-023-02094-w" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Elucidating the evolutionary process of animal adaptation to deserts is key to understanding adaptive responses to climate change. Here we generated 82 individual whole genomes of four fox species (genus Vulpes) inhabiting the Sahara Desert at different evolutionary times. We show that adaptation of new colonizing species to a hot arid environment has probably been facilitated by introgression and trans-species polymorphisms shared with older desert resident species, including a putatively adaptive 25 Mb genomic region. Scans for signatures of selection implicated genes affecting temperature perception, non-renal water loss and heat production in the recent adaptation of North African red foxes (Vulpes vulpes), after divergence from Eurasian populations approximately 78 thousand years ago. In the extreme desert specialists, Rueppell’s fox (V. rueppellii) and fennec (V. zerda), we identified repeated signatures of selection in genes affecting renal water homeostasis supported by gene expression and physiological differences. Our study provides insights into the mechanisms and genetic underpinnings of a natural experiment of repeated adaptation to extreme conditions.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="linderoth_genetic_2023" class="col-sm-8"> <div class="title">Genetic basis of aposematic coloration in a mimetic radiation of poison frogs</div> <div class="author"> Tyler Linderoth, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Diana Aguilar-Gómez, Emily White, Evan Twomey, Adam Stuckert, Ke Bi, Amy Ko, Natalie Graham, Joana L. Rocha, Jason Chang, Matthew D. MacManes, Kyle Summers, Rasmus Nielsen' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> Apr 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2023.04.20.537757"></span> <span class="__dimensions_badge_embed__" data-doi="10.1101/2023.04.20.537757" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The evolution of mimicry in a single species or population has rippling inter and intraspecific effects across ecological communities, providing a fascinating mechanism of phenotypic diversification. In this study we present the first identification of genes underlying Müllerian mimicry in a vertebrate, the Peruvian mimic poison frog, Ranitomeya imitator. We sequenced 124 R. imitator exomes and discovered loci with both strong divergence between different mimetic morphs and phenotypic associations within an intraspecific admixture zone, implicating mc1r, asip, bsn, retsat, and krt8.2 in the evolution of mimetic color phenotypes. We confirmed these associations for most candidate genes through linkage mapping in a lab-reared pedigree. We also sequenced transcriptomes from the model species, allowing tests for introgression and revealing that the mimetic resemblance between R. imitator and the models evolved independently. Selection analyses of the candidate genes show that the mimicry phenotypes likely have evolved through selective sweeps acting on polygenic variation. Our results suggest that the evolutionary origins and molecular mechanisms underlying mimicry phenotypes in vertebrates may be radically different from those previously documented in invertebrates such as the iconic Heliconius butterfly mimicry complex. One Sentence Summary Müllerian mimicry evolved through independent selective sweeps on color and pattern loci in the mimic poison frog.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/coverphoto_sable-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/coverphoto_sable-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/coverphoto_sable-1400.webp"></source> <img src="/assets/img/publication_preview/coverphoto_sable.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="coverphoto_sable.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="rocha_african_2022" class="col-sm-8"> <div class="title">African climate and geomorphology drive evolution and ghost introgression in sable antelope</div> <div class="author"> <em>Joana L. Rocha</em>, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Pedro Vaz Pinto, Hans R. Siegismund, Matthias Meyer, Bettine Vuuren, Luís Veríssimo, Nuno Ferrand, Raquel Godinho' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Molecular Ecology</em>, Apr 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.16427" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1111/mec.16427"></span> <span class="__dimensions_badge_embed__" data-doi="10.1111/mec.16427" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The evolutionary history of African ungulates has been explained largely in the light of Pleistocene climatic oscillations and the way these influenced the distribution of vegetation types, leading to range expansions and/or isolation in refugia. In contrast, comparatively fewer studies have addressed the continent’s environmental heterogeneity and the role played by its geomorphological barriers. In this study, we performed a range-wide analysis of complete mitogenomes of sable antelope (Hippotragus niger) to explore how these different factors may have contributed as drivers of evolution in southcentral Africa. Our results supported two sympatric and deeply divergent mitochondrial lineages in west Tanzanian sables, which can be explained as the result of introgressive hybridization of a mitochondrial ghost lineage from an archaic, as-yet-undefined, congener. Phylogeographical subdivisions into three main lineages suggest that sable diversification may not have been driven solely by climatic events affecting populations differently across a continental scale. Often in interplay with climate, geomorphological features have also clearly shaped the species’ patterns of vicariance, where the East Africa Rift System and the Eastern Arc Mountains acted as geological barriers. Subsequent splits among southern populations may be linked to rearrangements in the Zambezi system, possibly framing the most recent time when the river attained its current drainage profile. This work underlines how the use of comprehensive mitogenomic data sets on a model species with a wide geographical distribution can contribute to a much-enhanced understanding of environmental, geomorphological and evolutionary patterns in Africa throughout the Quaternary.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/desert_scheme-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/desert_scheme-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/desert_scheme-1400.webp"></source> <img src="/assets/img/publication_preview/desert_scheme.jpeg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="desert_scheme.jpeg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="rocha_life_2021" class="col-sm-8"> <div class="title">Life in Deserts: The Genetic Basis of Mammalian Desert Adaptation</div> <div class="author"> <em>Joana L. Rocha</em>, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Raquel Godinho, José C. Brito, Rasmus Nielsen' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Trends in Ecology &amp; Evolution</em>, Apr 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.tree.2021.03.007"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.tree.2021.03.007" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Deserts are among the harshest environments on Earth. The multiple ages of different deserts and their global distribution provide a unique opportunity to study repeated adaptation at different timescales. Here, we summarize recent genomic research on the genetic mechanisms underlying desert adaptations in mammals. Several studies on different desert mammals show large overlap in functional classes of genes and pathways, consistent with the complexity and variety of phenotypes associated with desert adaptation to water and food scarcity and extreme temperatures. However, studies of desert adaptation are also challenged by a lack of accurate genotype-phenotype-environment maps. We encourage development of systems that facilitate functional analyses, but also acknowledge the need for more studies on a wider variety of desert mammals.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="rocha_convergent_2021" class="col-sm-8"> <div class="title">Convergent evolution of increased urine‐concentrating ability in desert mammals</div> <div class="author"> <em>Joana L. Rocha</em>, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'José C. Brito, Rasmus Nielsen, Raquel Godinho' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Mammal Review</em>, Mar 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1111/mam.12244"></span> <span class="__dimensions_badge_embed__" data-doi="10.1111/mam.12244" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>1. One of the most celebrated textbook examples of physiological adaptations to desert environments is the unique ability that desert mammals have to produce hyperosmotic urine. Commonly perceived as an adaptation mainly observed in small rodents, the extent to which urine-concentrating ability has evolved independently in distinct mammalian lineages has not previously been assessed using modern phylogenetic approaches. 2. We review urine-concentrating ability data from the literature in 121 mammalian species with geographic ranges encompassing varying climatic conditions. We explicitly test the general hypothesis that desert-dwelling mammals have evolved greater ability to concentrate urine than non-desert species, controlling for body mass, phylogenetic affinity and other covariates. 3. Ancestral state reconstruction across our dataset’s phylogeny shows that the ability to produce hyperosmotic urine, measured as maximum urine osmolal-ity, has evolved convergently in mammalian species with geographic ranges characterised by low mean annual aridity index. 4. Phylogenetic generalised least-squares (PGLS) models show that the mean annual aridity index of a species’ geographic range largely predicts its urine-concentrating ability, even when accounting for body mass differences, phylogenetic correlations , the specific condition under which urine osmolality was measured, the method used to measure urine osmolality, and the species’ diet. 5. In contrast, we find much weaker correlations between mass-adjusted basal metabolic rate and environmental variables when analysing 84 of the species included in the urine osmolality analysis. 6. Taken together, our results not only show that desert mammals effectively concentrate more urine than non-desert mammals, but further suggest that aridity is likely to have been one of the main selective pressures leading to increasing maximum urine-concentrating ability and driving its repeated evolution in different desert mammalian lineages.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Joana L. Rocha. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js"></script> <script defer src="/assets/js/common.js"></script> <script defer src="/assets/js/copy_code.js" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>